front/components/vaults/VaultWebsiteModal.tsx

```diff
@@ -333,7 +334,7 @@ export default function VaultWebsiteModal({
     );
     setIsSaving(false);
     if (res.ok) {
-      await mutateVaultDataSourceViews();
+      void mutateVaultDataSourceViews();
```

Comment:
naive question: is there a need to mutate here, since apparently we router.push right below? (outside scope of this PR though)

Reply:
I'm no vercel expert but maybe sometime router.push() do not reload the page, just navigate to it within the app ? (but looking at the doc, it's called shallow routing and is opt-in)

Or it doesn't empty the cache ?